howtos/how-to-make-clickable-tabs-for-conversations/seed-users.js

```diff
@@ -0,0 +1,177 @@
+// THIS IS SETUP CODE FOR THE EXAMPLE
+// You won't need any of it in your live app!
+//
+// It's just here so that you can play around with this example more easily
+// When you run the script, we make sure the two example users are created,
+// recreate the two conversations, and send messages from the example users
+
+import fetch from "node-fetch";
+
+const appId = "<APP_ID>";
+const secretKey = "<SECRET_KEY>";
+
+const basePath = "https://api.talkjs.com";
+
+async function setupConversation(i) {
```

Comment:
Do you need the `i` parameter here? As I understand it this is a one-time setup, and you're only calling it with 1, so could you have `tabConversationExample1` etc. instead

Reply:
Done!